pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.isort]
 skip_glob = ["*/deps/*", "*/build/*", "*/obselete/*"]
-known_third_party = ["PIL", "attr", "conf", "cv2", "demo_runner", "hypothesis", "magnum", "matplotlib", "numba", "numpy", "pytest", "quaternion", "scipy", "settings", "setuptools", "torch", "torchvision", "tqdm"]
+known_third_party = ["PIL", "attr", "conf", "cv2", "demo_runner", "git", "hypothesis", "magnum", "matplotlib", "numba", "numpy", "pytest", "quaternion", "scipy", "settings", "setuptools", "torch", "torchvision", "tqdm"]
```

Comment:
Not sure why pre-commit forced this change. Couldn't commit without it.

Reply:
That's fine/expected.  This line just gets updated overtime.